modules/websocket/doc_classes/WebSocketServer.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="WebSocketServer" inherits="WebSocketMultiplayerPeer" category="Core" version="3.1">
+	<brief_description>
+		A WebSocket server implementation
+	</brief_description>
+	<description>
+		This class implements a WebSocket server that can also support the high level multiplayer API.
+		After starting the server ([method listen]), you will need to [method NetworkedMultiplayerPeer.poll] it at regular intervals (e.g. inside [method Node._process]). When client connects, disconnects, or send data, you will receive the appropriate signal.
```

Comment:
When clients connect, disconnect or send data, you will receive the appropriate signal.